LiteFarmOrg/LiteFarm
# [LF-2530, LF-2454] Addition of the soil water potential reading to the farm map page and addition of sensor type validation in the add readings API

To Test:

1. Add sensors to the farm.
2. make sure the webhook works and the sensor_readings table have data.
3. go to the farm map page. 
4. long click on the sensor and check the latest readings of temperature and soil water potential.

For more details: 
https://lite-farm.atlassian.net/browse/LF-2530
Please connect with me, if you have any questions.
